platform/config/config-reader-yaml/src/test/java/org/codice/ddf/config/YamlConfigReaderTest.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.config;
+
+import java.io.File;
+import java.net.URL;
+import java.util.Set;
+import org.codice.ddf.config.model.impl.RegistryConfigImpl;
+import org.codice.ddf.config.reader.impl.YamlConfigReaderImpl;
+import org.junit.Test;
+
+public class YamlConfigReaderTest {
+
+  @Test
```

Comment:
Ooops....This needs to be removed.

Reply:
You made me look bad since you were the only one with actual test cases 😄 

❓ Why remove it?